client/src/components/Details.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReadMore from './ReadMore'
```

Comment:
I see you're importing the ReadMore module here but I don't see where it's being used in the file. Is it necessary to import in at this time?